From: SeongJae Park <sj@kernel.org>
Date: Wed, 7 Jan 2026 17:44:01 -0800
Subject: [PATCH] ==== code cleanups ====

mm/damon: cleanup kdamond, damon_call(), damos filter and DAMON_MIN_REGION

Do miscellaneous code cleanups for improving readability.  First three
patches cleanup kdamond termination process, by removing unused
operation set cleanup callback (patch 1) and moving damon_ctx specific
resource cleanups on kdamond termination to synchronization-easy place
(patches 2 and 3).  Next two patches touch damon_call() infrastructure,
by refactoring kdamond_call() function to do less and simpler locking
operations (patch 4), and documenting when dealloc_on_free does work
(patch 5).  Final three patches rename things for clear uses of those.
Those rename damos_filter_out() to be more explicit about the fact that
it is only for core-handled filters (patch 6), DAMON_MIN_REGION macro to
be more explicit it is not about number of regions but size of each
region (patch 7), and damon_ctx->min_sz_region to be different from
damos_access_patern->min_sz_region (patch 8), so that those are not
confusing and easy to grep.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus__4whrt88 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus__4whrt88

diff --git a/hkml_cv_bogus/hkml_cv_bogus__4whrt88 b/hkml_cv_bogus/hkml_cv_bogus__4whrt88
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.47.3

